File: jdroid-android-sample/src/main/java/com/jdroid/android/sample/ui/rateme/RateAppActivity.java
package com.jdroid.android.sample.ui.rateme;

import androidx.fragment.app.Fragment;

import com.jdroid.android.activity.FragmentContainerActivity;

public class RateAppActivity extends FragmentContainerActivity {

	@Override
	protected Class<? extends Fragment> getFragmentClass() {
		return RateAppFragment.class;
	}
}

File: jdroid-android-sample/src/main/java/com/jdroid/android/sample/ui/rateme/RateAppActivity.kt
package com.jdroid.android.sample.ui.rateme

import androidx.fragment.app.Fragment

import com.jdroid.android.activity.FragmentContainerActivity

class RateAppActivity : FragmentContainerActivity() {

    override fun getFragmentClass(): Class<out Fragment> {
        return RateAppFragment::class.java
    }
}
